cmd/algorand-indexer/main.go

```diff
@@ -115,6 +115,7 @@ func init() {
 
 	rootCmd.AddCommand(importCmd)
 	importCmd.Hidden = true
+	daemonCfg.setFlags(daemonCmd.Flags())
```

Comment:
I think daemonCfg should be initialized in a function which also sets this and defines the flags. We keep this code in close proximity across all our apps, so I think its important to do here as well.

Reply:
Put everything in a single function in the most recent changes.